## Caroline Choiniere
 B.S. Environmental Science <br/>
 Southern New Hampshire University <br/>
 Email: caroline.choiniere@snhu.edu <br/>
 [Resume](https://github.com/caroline-choiniere/caroline-choiniere.github.io/blob/367007f269d4ef6fc9d3925fd4a1d8932b7957d4/Resume%20doc%20v.pdf) <br/>

<img src= "IMG_9034.jpeg" width="300" align="left">

## About Me
 
As a child, I would bring insects to my wary mother and she would always say, “Caroline you are either going to be an entomologist or a botanist.” I was given an echeveria amoena succulent when I was seven and knew at that moment what I wanted to do with my future.
 
I am an experienced plant retail merchandiser known for communication, problem solving, and detail driven analysis. I have developed a deep interest in plant-insect interactions as well as helping customers design native plant landscapes. A highlight of my day is seeing the hummingbird clearwing hawkmoths as well as other Lepidoptera species. 
 
I am an aspiring environmental science student at Southern New Hampshire University with a focus on combining environmental science and horticulture in my personal and professional life.

## Sample Work Products
### Undergraduate Research Day 2024
Check out [my undergraduate research day 2024 poster](https://hdl.handle.net/10474/3833) from Spring 2024!
Presented research examining the structure of the sod farming industry as research needs to be done to determine how the regulation of sod crop affects invasive plant and insect species.

## Palmers Penguins [Analysis here](https://caroline-choiniere.github.io/BioStat/PPHW3.html)
Creating and rendering a Quarto document containing an analysis on the palmerpenguins data into a reproducible report. Then making a local repository in GitHub.

## Extinct Plants Dataset Breakdown [Analysis here](https://caroline-choiniere.github.io/BioStat/ExtPlaHW4.html)
The data being worked with is from the International Union for Conservation of Nature (IUNC) Red List of Threatened Species (Version 2020-1). Florent Lavergne scrapped and prepared the Plants in Danger project to develop an info-graphic showing the biodiversity crisis plants are undergoing around the world. The data set is being used to explore the link between socio-economic activity and plant extinction. Determining if extinction risk varies by region is another potential use of the data set. Africa and Madagascar have the highest count of extinct plant species per Continent and Country with links to socio-economic disturbance in the forms of over-industrialization of green areas and disruption of natural forest (Feintrenie, 2014; Ralimanana et al., 2022). It is shown in IUCN data that over-exploitation and unsustainable agricultural practices in Madagascar are a threat to biodiversity (Stévart et al., 2019). Plant extinction varies by Continent, and a link to social or economic activity is found in Africa and Madagascar.

IUCN (2020) The IUCN Red List of Threatened Species. Version 2020-1. https://www.iucnredlist.org

## RNA Sequencing Part 1 [Analysis here](https://caroline-choiniere.github.io/BioInformatics/RNAS1.3.html)
Learned how to use edgeR to analyze r-seq data and to compare RNA-seq data from two treatments of dp. I was able to use the final differential expression analysis and see the genes with the highest measure of differences between the two samples. Using flybase was interesting and gave further info on what each gene did and why or why not it was differentially regulated between each instar.

## RNA Sequencing Part 2 [Analysis here](https://caroline-choiniere.github.io/BioInformatics/RNAsedII.at.html)
In the first part, we learned how to analyze RNA-seq data and determine which genes in the sample data-set were differentially expressed or turned on/off from instar 1 to instar 2. We used edgeR alongside a couple other R packages to further analyze the sample data set.

RNA-seq part 2 is focusing on visualizing data and narrowing down on genes that are different as well as analysis of the patterns in the data. This part is focusing on a sample data set from the R bioinformatics cookbook using the model plant species Arabidopsis thaliana. This is focusing on how its gene expression varies in different parts of the plant. Then we can use RNA-seq to see if gene expression varies by ecotype.

## Phylogenetics with R [Analysis here](https://caroline-choiniere.github.io/BioInformatics/PAVhw9.1.html)
Learned how to analyze data to build phylogenies; and phylogentic trees will be made using genetic data and big datasets. ape and treeio packages will be used to create formats of evolutionary trees, then playing with ggtree to see how to create different visuals of evolutionary trees. The final goal is to interpret the output of the programs and read an evolutionary tree. This exercise uses many basics from ggplot and I will have to try some formatting and display alternatives in future projects to see what changes in the tree visuals.









